src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonItem } from './PersonItem';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const findPerson = (name: string | null) => {
+    return people.find(person => person.name === name) || null;
+  };
```

Comment:
```suggestion
  const findPersonParent = (parentName: string | null) => {
    return people.find(({ name })=> name === parentName);
  };
```

Reply:
I'm not searching particular parrent, I'm just searching person by name